### PR TITLE
✨ Add legend to cloropleth static component

### PR DESCRIPTION
**Pull request by enzoggqs**
## Motivation

Create a legend to map colors

## Changes

Add a legend component in the application

- Something that added/fixed/changed

## Status Checklist

- [ ] Test

## Screenshots

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,11 +34,12 @@
     "react-icons": "^4.3.1",
     "react-popper": "^2.2.5",
     "react-router-dom": "^5.3.0",
-    "react-scripts": "4.0.3",
+    "react-scripts": "4.0.1",
```
Comment (LeoSilvaGomes) [inline package.json:37]: Why did you need to downgrade the lib?

File: src/components/InformationsBar/SummaryCard/styles.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { Card } from '@mui/material';
 import styled from 'styled-components';
 
 export const Container = styled(Card)`
+  top: 150px;
   position: fixed;
   z-index: 2;
```
Comment (LeoSilvaGomes) [inline src/components/InformationsBar/SummaryCard/styles.ts:7]: Try to add this z-index on `InformationsBar` styles, should work well there too, so you can remove this from here and here https://github.com/gpmc-lab-ufrgs/atlas/pull/34/files?diff=unified&w=0#:~:text=position%3A%20fixed%3B-,z%2Dindex%3A%202%3B,-font%2Dsize%3A%2014px

File: src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        <defs>
+          <linearGradient id="myGradient">
+            <stop offset="25%" stopColor="#ADDC91" />
+            <stop offset="50%" stopColor="#6CC24A" />
+            <stop offset="75%" stopColor="#509E2F" />
+            <stop offset="100%" stopColor="#4A7729" />
+          </linearGradient>
+        </defs>
+      </svg>
+      <Box paddingTop="25px" px="25px">
```
Comment (LeoSilvaGomes) [inline src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx:20]: Add some spaces here to improve the readability, even this is good for code live

File: src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      <svg style={{ height: 0 }}>
+        <defs>
+          <linearGradient id="myGradient">
+            <stop offset="25%" stopColor="#ADDC91" />
+            <stop offset="50%" stopColor="#6CC24A" />
+            <stop offset="75%" stopColor="#509E2F" />
+            <stop offset="100%" stopColor="#4A7729" />
+          </linearGradient>
+        </defs>
+      </svg>
```
Comment (LeoSilvaGomes) [inline src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx:19]: This need to be here, maybe you can segregate in other component, but if can be a internal component... as here https://github.com/gpmc-lab-ufrgs/atlas/blob/master/src/components/Sidebar/RegionDetails/ComparisonSection.tsx#:~:text=const%20ComparisonResult%20%3D,)%3B

File: src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      <Box paddingTop="25px" px="25px">
+        <Styles.Title>População</Styles.Title>
+      </Box>
+      <Box sx={{ dispay: 'flex', alignItem: 'center' }}>
```
Comment (LeoSilvaGomes) [inline src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx:23]: Add a break line here too

**Review by LeoSilvaGomes — APPROVED** [review]
Great job 🎉

File: package.json (hunk 1 above)
Comment (enzoggqs) [reply package.json:37]: I don't remember now, but I think I was facing a bug and forgot to remove it 😅.  I'll remove.

File: src/components/InformationsBar/SummaryCard/styles.ts (hunk 2 above)
Comment (enzoggqs) [reply src/components/InformationsBar/SummaryCard/styles.ts]: Done

File: src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx (hunk 4 above)
Comment (enzoggqs) [reply src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx]: Done!

File: src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx (hunk 5 above)
Comment (enzoggqs) [reply src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx:15]: Fixed!

File: src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx (hunk 3 above)
Comment (enzoggqs) [reply src/components/InformationsBar/CloroplethLegend/CloroplethLegend.tsx:12]: Fixed!

**Review by LeoSilvaGomes — APPROVED** [review]
🎉

**Review by valdersonjr — APPROVED** [review]
Nice work!
